docs/service-loadbalancer.md

```diff
@@ -306,14 +306,8 @@ kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.13.11/conf
 The commands will deploy MetalLB version 0.13.11 into Namespace
 `metallb-system`. You can also refer to this [MetalLB installation
 guide](https://metallb.universe.tf/installation) for other ways of installing
-MetalLB.
-
-As MetalLB will allocate external IPs for all Services of type LoadBalancer,
```

Comment:
Even though it is pretty obvious, I would still keep a sentence like this one here:

> To avoid conflicts, make sure not to use the `service.antrea.io/external-ip-pool` annotation (Antrea `ServiceExternalIP` feature) when using MetallLB to allocate LoadBalancer IPs.

Reply:
added.